school_attendance/wizard/monthly_attendance_wizard_view.xml

```diff
@@ -9,25 +9,44 @@
                 <sheet>
                     <group colspan="4">
                         <group>
-                            <field name="academic_year_id" required="1"
-                                 options="{&quot;no_create&quot;: True, &quot;no_search&quot;: True, &quot;no_open&quot;: True}"/>
-                            <field name="month" required="1"/>
-                            <field name="month_str" required="1" invisible="1"/>
+                            <field
+                                name="academic_year_id"
+                                required="1"
+                                options="{&quot;no_create&quot;: True, &quot;no_search&quot;: True, &quot;no_open&quot;: True}"
```

Comment:
**no_search**  This is the first time I have encountered this attribute in my experience with Odoo. :red_circle: